Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt

```diff
@@ -140,21 +140,23 @@ fun JetsnackBottomBar(
                     }
                 )
 
+                val text = stringResource(section.title).uppercase(
```

Comment:
Unrelated: Should this be `remember`ed?

Reply:
You wouldn't be able to remember the CompositionLocal so all you'd be remembering is the uppercase really